docs/api/Makefile.am

```diff
@@ -14,7 +14,7 @@ DOC_MAIN_SGML_FILE=$(DOC_MODULE)-docs.sgml
 # The directory containing the source code. Relative to $(srcdir).
 # gtk-doc will search all .c & .h files beneath here for inline comments
 # documenting the functions and macros.
-DOC_SOURCE_DIR=../../gusb
+DOC_SOURCE_DIR=$(top_srcdir)/gusb $(top_builddir)/gusb
```

Comment:
I think $(top_srcdir)/gusb should do it, or do we need to parse any generated file?